modules/components/pages/processors/aws_bedrock_embeddings.adoc

```diff
@@ -0,0 +1,164 @@
+= aws_bedrock_embeddings
+// tag::single-source[]
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates vector embeddings from text prompts, using the https://aws.amazon.com/bedrock/[AWS Bedrock API^].
+
+ifndef::env-cloud[]
+Introduced in version 4.37.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+Advanced::
++
+--
+```yml
+# All config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  region: ""
+  endpoint: ""
+  credentials:
+    profile: ""
+    id: ""
+    secret: ""
+    token: ""
+    from_ec2_role: false
+    role: ""
+    role_external_id: ""
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+======
+
+This processor sends text prompts to your chosen large language model (LLM) which generates vector embeddings for them, using the AWS Bedrock API.
+
+For more information, see the https://docs.aws.amazon.com/bedrock/latest/userguide[AWS Bedrock documentation^].
+
+== Fields
+
+=== `model`
```

Comment:
I moved the model and text fields to the top for users who only want to use the basic config.